src/components/dashboard/jumbotron/CurrentJumbotron.js

```diff
@@ -52,7 +52,7 @@ export default class CurrentJumbotron extends Component {
    * @returns {*} HTML for current events
    */
   getCurrentDiv() {
-    if (this.state.hasCurrEvent) {
+    if (this.hasCurrEvent()) {
```

Comment:
ommmmgggg how did I miss this?! So sorry... Thanks for the catch

Reply:
No problemo!